capture-sdk/sdk/src/main/java/net/gini/android/capture/Amount.java

```diff
@@ -0,0 +1,35 @@
+package net.gini.android.capture;
+
+import androidx.annotation.NonNull;
+
+import java.math.BigDecimal;
+
+public class Amount {
+
+    private static Amount mAmount;
+
+    @NonNull
+    private final BigDecimal mBigDecimal;
+
+    @NonNull
+    private final AmountCurrency mCurrency;
+
+
+
+    public Amount(@NonNull final BigDecimal decimal, @NonNull final AmountCurrency currency) {
+        mBigDecimal = decimal;
+        mCurrency = currency;
+    }
+
+
+    public String amountToPay() {
```

Comment:
[Request] I would prefer to avoid using double and instead use a DecimalFormat to format the number. Something like this:
```
final DecimalFormat decimalFormat = new DecimalFormat("0.00", DecimalFormatSymbols.getInstance(Locale.ENGLISH));
decimalFormat.setParseBigDecimal(true);
return decimalFormat.format(mBigDecimal) + ":" + mCurrency.name();
```